pyiron_base/jobs/job/generic.py

```diff
@@ -50,6 +50,7 @@
 from pyiron_base.jobs.job.extension.server.generic import Server
 from pyiron_base.database.filetable import FileTable
 from pyiron_base.interfaces.has_dict import HasDict
+from pyiron_base.storage.datacontainer import DataContainer
```

Comment:
```suggestion
```
Not used.

Reply:
The `DataContainer` is still used in line 46.